frontend/src/views/funding/FundingOverviewView.vue

```diff
@@ -54,47 +163,211 @@
+import { FUNDING_AGREEMENT_STATUS_CODES } from '@/utils/constants'
+import { mapState } from 'pinia'
+import { useAuthStore } from '@/stores/auth'
+import { ApiRoutes } from '@/utils/constants'
+import ApiService from '@/common/apiService'
+import moment from 'moment'
 
 export default {
   name: 'FundingOverviewView',
-  components: { AppBackButton, OrganizationHeader },
+  components: { AppButton, AppBackButton, AppLabel, AppButtonRadioGroup, OrganizationHeader },
   data() {
     return {
+      format,
       tab: null,
-      fundingAgreements: [
-        {
-          guid: '21d0d405-900c-ef11-9f89-000d3af44283',
-          faNumber: 'FA-123456',
-          faType: 'Base Funding',
-          facility: 'Facility 1',
-          startDate: 'Nov 22, 2023',
-          endDate: 'Nov 22, 2024',
-          status: 'FA Signature Pending',
-        },
-        {
-          guid: '21d0d405-900c-ef11-9f89-000d3af44283',
-          faNumber: 'FA-000002',
-          faType: 'Supplementary Allowances',
-          facility: 'Facility 2',
-          startDate: 'May 15, 2023',
-          endDate: 'May 15, 2024',
-          status: 'FA Submitted to Ministry',
-        },
-      ],
+      loading: false,
+      fundingAgreements: [],
+      orgUsers: [],
+      selectedFacility: null,
+      selectedDateFilterType: null,
+      selectedPaymentFilterTypes: [],
+      startDateFrom: null,
+      startDateTo: null,
+      menuStartDateFrom: false,
+      menuStartDateTo: false,
       headers: [
-        { title: 'FA Number', key: 'faNumber' },
-        { title: 'FA Type', key: 'faType' },
-        { title: 'Facility', key: 'facility' },
+        { title: 'Funding Agreement Number', key: 'fundingAgreementNumber' },
+        { title: 'Funding Agreement Type', key: 'fundingAgreementType' },
+        { title: 'Facility Name', key: 'facility' },
+        { title: 'Signing Expense Authority', key: 'expenseAuthorityName' },
         { title: 'Start Date', key: 'startDate' },
         { title: 'End Date', key: 'endDate' },
         { title: 'Status', key: 'status' },
         { title: 'Actions', key: 'actions' },
       ],
+      dateFilterTypes: [
+        { label: '3 Months', value: '3Months' },
+        { label: '6 Months', value: '6Months' },
+        { label: 'YTD', value: 'YTD' },
+        { label: 'Custom', value: 'Custom' },
+      ],
+      paymentFilterTypes: [
+        { label: 'Base Funding', value: 'baseFunding' },
+        { label: 'Supplementary Allowances', value: 'supplementaryAllowances' },
+        { label: 'Other', value: 'other' },
+      ],
+    }
+  },
+  computed: {
+    ...mapState(useAuthStore, ['userInfo']),
+    formattedStartDateFrom() {
+      return this.startDateFrom ? format.formatDate(this.startDateFrom) : ''
+    },
+    formattedStartDateTo() {
+      return this.startDateTo ? format.formatDate(this.startDateTo) : ''
     }
   },
+  async created() {
+    this.FUNDING_AGREEMENT_STATUS_CODES = FUNDING_AGREEMENT_STATUS_CODES
```

Comment:
You don't need to define this in created() unless it's required in the template.

Reply:
Fixed.